### PR TITLE
Removed nondbMakeFile

### PR DESCRIPTION
**Pull request by brijesh-amin**
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## Problem/Feature

There are two make files with almost identical content. I have removed nondbMakeFile to provide more flexibility in the future and easier templating.

## Description of Changes: 

- Targets Merge Makefile templates #272

## Checklist

- [x] I have self-reviewed the changes being requested
- [x] I have updated the documentation (check issue ticket #218)

### CODE REVIEW

**Review by Ujstor — CHANGES_REQUESTED** [review]
Tailwind and Htmx flags are connected...if Tailwind is selected, Htmx templates will also be generated. In that case, we need to add '@templ generate' to the Makefile build.

File: cmd/template/framework/files/makefile.tmpl [hunk 1]
```diff
@@ -5,14 +5,15 @@ all: build
 
 build:
 	@echo "Building..."
-	{{if .AdvancedOptions.htmx}}@templ generate{{end}}
+	{{if or ( .AdvancedOptions.htmx ) ( .AdvancedOptions.tailwind )}}@templ generate{{end}}
```
Comment (brijesh-amin) [inline cmd/template/framework/files/makefile.tmpl:8]: @Ujstor modified to add `@templ generate` when tailwind is selected

**Review by Ujstor — APPROVED** [review]
LGTM

**Comment by Ujstor** [comment]
@brijesh-amin My mistake, I merged it, but you added Submodule xs. If you won't ypu can open a PR and remove the linked folder

**Comment by brijesh-amin** [comment]
> @brijesh-amin My mistake, I merged it, but you added Submodule xs. If you won't you can open a PR and remove the linked folder

My bad I was testing and did not realize xs got added. Will remove it now.

**Pull request merged**